extensions/leveldb/runtime/src/main/doc/limitations.adoc

```diff
@@ -0,0 +1,5 @@
+In native mode extension uses port of the LevelDB written in Java (https://github.com/dain/leveldb#leveldb-in-java[documentation]),
+which is within 10% of the performance of the C++ original. Usage of native implementation will be investigated as a new issue.
+
+GraalVM does not support `ObjectOutputStream.writeObject()` (https://github.com/oracle/graal/issues/460[issue]).
+In native mode extension uses Jackson serialization instead. This approach could have some performance impact.
```

Comment:
Looking at [this comparison from 2014](http://rick-hightower.blogspot.com/2014/04/which-is-faster-java-object.html) the impact will most likely be be positive in most cases. I'd remove the whole paragraph.

Reply:
I didn't know that, but it is nice. I'll remove the comment

Reply:
done